src/utils/authFetch.js

```diff
@@ -10,10 +10,18 @@ export const authFetch = (url, data) => {
     return fetch(url, authData).then(response => {
         if(response.status >= 200 && response.status < 300)
             return response
-        else
+        else {
```

Comment:
Tu może pójść tak naprawdę 500, więc nie jestem pewny, czy tym elsem bym to robił. Zrobiłbym bardziej else if (response.status == 401) (trzeba sprawdzić, ale to jest chyba unauth)